UnityProject/Assets/Resources/Prefabs/SceneConstruction/NestedManagers/ScriptableObjectSingletons.prefab

```diff
@@ -57,7 +57,6 @@ MonoBehaviour:
     type: 2}
   CommonPrefabs: {fileID: 11400000, guid: bf1ea2a89b19143c996995d650266ff2, type: 2}
   CommonCooldowns: {fileID: 11400000, guid: 5b45f6c0f9827844aa17cd49d2e888f1, type: 2}
-  AmmoPrefabs: {fileID: 11400000, guid: 80457940d6023324eb9a2f1fc72d6a49, type: 2}
```

Comment:
`AmmoPrefabs` was removed. Is that intended?

Reply:
yeah, accidental

Reply:
looks like it was removed in a previous PR so the prefab wasn't updated